release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -521,6 +521,68 @@ If the following conditions are true, the feature is enabled by default:
 [id="ocp-4-15-monitoring"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-15-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for in-cluster monitoring stack components and dependencies:
+
+* Alertmanager to 0.26.0
+* kube-state-metrics to 2.10.1
+* node-exporter to 1.7.0
+* Prometheus to 2.48.0
+* Prometheus Adapter to 0.11.2
+* Prometheus Operator to 0.70.0
+* Thanos Querier to 0.32.5
+
+[id="ocp-4-15-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *Changed*
+** Modified the `NodeClockNotSynchronising` alerting rule to also consider the PTP protocol.
+
+[id="ocp-4-15-monitoring-new-metrics-server-to-access-metrics-API-technology-preview"]
+==== New Metrics Server component to access the Metrics API (Technology Preview)
+This release introduces a Technology Preview option to add a Metrics Server component to the in-cluster monitoring stack. 
+If enabled, Metrics Server, rather than Prometheus Adapter, collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs.
```

Comment:
It might be unclear if you need to enable the metrics server explicitly or not. In practice, metrics server is automatically installed instead of Prometheus adapter when the FeatureGate CR is configured with `TechPreviewNoUpgrade`.

https://docs.openshift.com/container-platform/4.14/nodes/clusters/nodes-cluster-enabling-features.html

Reply:
Added verbiage about how Metrics Server is installed.